cmd/ethermintd/ledger.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/accounts/usbwallet"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/tharsis/ethermint/ethereum/rpc/backend"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/libs/log"
+	rpctypes "github.com/tharsis/ethermint/ethereum/rpc/types"
+)
+
+const (
+	flagIndex = "index"
+)
+
+func runAddCmd(cmd *cobra.Command, _ []string) error {

```

Comment:
is this stored in the keyring?

Reply:
no further information is not stored in the keyring for ledger